device/arista/x86_64-arista_7800_sup/globaldb.conf

```diff
@@ -0,0 +1,2 @@
+start_global_db=1
```

Comment:
How can the global db entries be flushed by swss on startup. (swss.sh for reference)

Reply:
we have switch-id as part of key for entries in global db, that info can be used to flush the db, but the flush function should be handle by swss code not in the scope of this PR.